sources/mongodb/requirements.txt

```diff
@@ -0,0 +1,2 @@
+pymongo>=4.4.1
```

Comment:
I think a version from like 6 months ago also will work? does not make sense to put the newest dep